server/core/src/irods_signal.cpp

```diff
@@ -23,19 +24,20 @@ void stacktrace_signal_handler(int _signal)
         std::exit(_signal);
     }
 
-    const auto max_path_length = 512;
+    // "_POSIX_PATH_MAX" is guaranteed to be defined as 256.
+    // This value represents the minimum path length supported by POSIX operating systems.
```

Comment:
no, minimum is correct. PATH_MAX exists, but isn't guaranteed to be defined and even if it is, it can be different across OS.

Reply:
the alternative is to look up the system limit using `fpathconf(stdin, _PC_PATH_MAX)`.

the issue is that it might be indeterminate. now we're back to either defining our value (which could result in `ENAMETOOLONG` errors) or using the guaranteed minimum of 256.

Reply:
portable seems good.  256 should be enough, and if not, that means someone has a non-package install way deep in their filesystem.   probably would cause other problems as well at that depth.

Reply:
agreed.